### ISSUE TITLE
eth_gasLimit method not implemented

### ISSUE THREAD

**Issue by 4gn3s** [ethereum/pyethapp#99]
Hello, when using _new_abi_contract_ from rpc_client to create a new _ABIContract_ object, and later calling a contract method, I get the following error: 

> {"id":5,"jsonrpc":"2.0","error":{"code":-32601,"message":"eth_gasLimit method not implemented"}}

I'm using geth 1.3.3. as a client.
In the [JSON RPC](https://github.com/ethereum/wiki/wiki/JSON-RPC) docs, _eth_gasLimit_ is indeed not defined. I guess it was replaced with [eth_getBlockByHash](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getblockbyhash), which has a gasLimit field.

Maybe both calls to _eth_gasLimit_ in rpc_client should be replaced with a call to _eth_estimateGas_, or with a series of two calls: _eth_blockNumber_ followed by _eth_getBlockByNumber_, if you want to stick to the gas limit convention?

I will gladly contribute if one of the suggested fixes is a correct solution.

**Comment by heikoheiko** [comment]
> I'm using geth 1.3.3. as a client.

?

**Comment by heikoheiko** [comment]
ah, so this is an issue in 
https://github.com/ethereum/pyethapp/blob/develop/pyethapp/rpc_client.py

**Comment by heikoheiko** [comment]
Btw: there is also this project
https://github.com/ConsenSys/ethjsonrpc

**Comment by 4gn3s** [comment]
Yeah, **eth_gasLimit** is called in the following lines: [178](https://github.com/ethereum/pyethapp/blob/develop/pyethapp/rpc_client.py#L178) and [199](https://github.com/ethereum/pyethapp/blob/develop/pyethapp/rpc_client.py#L199).
